.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Pull requests into js-ecqm-engine require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
```

Comment:
`js-ecqm-engine` -> `cqm-models`

Reply:
Good catch. Thought I'd fixed that when copy/pasting, evidently not.